src/handlers/int-test-support/handler.test.ts

```diff
@@ -0,0 +1,71 @@
+import { Context } from "aws-lambda";
+import { handler, TestSupportEvent } from "./handler";
+import { getS3Object, listS3Objects } from "./helpers/s3Helper";
+
+jest.mock("./helpers/s3Helper.ts");
+const mockedGetS3Object = getS3Object as jest.MockedFunction<
+  typeof getS3Object
+>;
+const mockedListS3Objects = listS3Objects as jest.MockedFunction<
+  typeof listS3Objects
+>;
+
+let givenContext: Context;
+let inputEvent: TestSupportEvent;
+const oldConsoleError = console.error;
+
+beforeEach(() => {
+  jest.resetAllMocks();
+
+  console.error = jest.fn();
+
+  givenContext = {
+    someContextItem: "some context item",
+  } as any;
+});
+
+afterAll(() => {
+  console.error = oldConsoleError;
+});
```

Comment:
Is it ok to do these things at top level? I've only ever seen them nested in `describes`, will they be scoped to this file?

Reply:
Yeah, I think they are, but I would rather have them in describes as well. Not sure if that's the scope of my work here though.